0002-crypto-conditions/0002-crypto-conditions.md

```diff
@@ -92,78 +92,79 @@ Crypto-conditions provide a mechanism to describe a signed message such that mul
 
 --- middle
 
-# Introduction {#intro}
-This specification describes a message format for defining distributable event descriptions (crypto-conditions) and the cryptographically verifiable event messages (fulfillments) that can be used to prove that the event occurred.
+# Introduction
+This specification describes a message format for crypto-conditions and fulfillments, with binary and string-based encodings for each.
 
-The specification defines both binary and string-based encoding for the messages.
+Crypto-conditions are **distributable event descriptions**. This means crypto-conditions say how to recognize a message without saying exactly what the message is. You can transmit a crypto-condition freely, but you cannot forge the message it describes. For convenience, we hash the description so that the crypto-condition can be a fixed size.
 
-## Terminology {#terminology}
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119].
+Fulfillments are **cryptographically verifiable messages** that prove an event occurred. If you transmit a fulfillment, then everyone who has the condition can agree that the condition has been met.
 
-Within this specification, the term "condition" refers to the hash of a description of a signed message.
+In the Interledger protocol, crypto-conditions and fulfillments provide irrepudiable proof that a transfer occurred in one ledger, as messages that can be easily shared with other ledgers. This allows ledgers escrow funds or hold a transfer conditionally, then execute the transfer automatically when the ledger sees the fulfillment of the stated condition.
 
```

Comment:
I was on the fence about this. I decided it was worthwhile to provide the context about how CCs fit into ILP in one paragraph of the intro. The spec already says it's part of the interledger project, so I think it's important to know why.

That doesn't preclude CCs from being useful elsewhere, of course. Maybe we should add an explicit statement to that effect.